indexing-hadoop/src/main/java/io/druid/indexer/JobHelper.java

```diff
@@ -588,6 +608,20 @@ public static Path makeSegmentOutputPath(
     return new Path(prependFSIfNullScheme(fileSystem, basePath), String.format("./%s", segmentDir));
   }
 
+  //This method is very similar to makeSegmentOutputPath() method except it returns output path upto version
```

Comment:
`makeSegmentOutputPath` appears to be unused now, why not just replace it with this one?